packages/location-management/src/LocationUnitModule.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
@ashfahan did you figure out how to work the nested routes, and if so is this code being used in the client?

Reply:
Not really, haven't worked over nested routing again and yes this code is unusable. should we remove it ?

Reply:
@ashfahan yes please remove it, you can create an issue to figure this out, if one does not yet exist